src/services/jwt.service.js

```diff
@@ -0,0 +1,38 @@
+const jwt = require('jsonwebtoken');
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
```

Comment:
Ensure that `process.env.JWT_KEY` is correctly set in your environment to securely sign and verify JWT access tokens.